luaui/Widgets/unit_smart_select.lua

```diff
@@ -68,7 +68,7 @@ for udid, udef in pairs(UnitDefs) do
 		ignoreUnits[udid] = true
 	end
 
-	local isMobile = (udef.canMove and udef.speed > 0.000001)  or  (includeNanosAsMobile and (udef.name == "armnanotc" or udef.name == "cornanotc" or udef.name == "legnanotc" or udef.name == "armnanotct2" or udef.name == "cornanotct2" or udef.name == "legnanotct2"))
+	local isMobile = (udef.canMove and udef.speed > 0.000001)  or  (includeNanosAsMobile and udef.customParams.isnanoturret)
```

Comment:
Maybe the customparam is redundant?
```suggestion
	local isMobile = not udef.isImmobile or (includeNanosAsMobile and (udef.isStaticBuilder and not udef.isFactory))
```

Reply:
Tested your suggestion and it seems to work as intended. I've removed the custom params again, they didn't work as I expected either.

This is also more in line with what Watch_The_Fort wanted on Discord; Using general params instead of explicit params to define the nano.

I am unsure about changing `(udef.canMove and udef.speed > 0.000001)` -> `not udef.isImmobile` as I don't want this new check to be the reason it breaks, but the testing I did showed no issues. Nanos are being grouped with mobile builders, same behavior as with explicit name checking.